apidoc/Identity.yml

```diff
@@ -141,6 +142,10 @@ methods:
 
   - name: isSupported
     summary: Determines if the current device supports Touch ID or Face ID.
+    description: | 
+        This module is only supported on Android 6.0 or newer. So, this method always returns
+        'false' for the older versions of the android.
+
```

Comment:
The grammar isn't quite right. Please change...
`...for the older versions of the android.`

...to...
`...on older versions of Android.`